### PR TITLE
添加pooling的sse实现

### PR DESCRIPTION
**Pull request by shitouren1994**
测试模型：

class TestPoolPadding(torch.nn.Module):
    def __init__(self):
        super(TestPoolPadding, self).__init__()
        self.max3x3s2p1 = torch.nn.MaxPool2d(kernel_size=3, stride=2, padding=1)
        self.max3x3s1p1 = torch.nn.MaxPool2d(kernel_size=3, stride=1, padding=1)
    def forward(self, x):
        x1 = self.max3x3s1p1(x)
        x2 = self.max3x3s2p1(x)
        return x1, x2

class TestPool(torch.nn.Module):
    def __init__(self):
        super(TestPool, self).__init__()
        self.avgpool_2x2s2 = torch.nn.AvgPool2d(kernel_size=2, stride=2)
        self.avgpool_3x3s2 = torch.nn.AvgPool2d(kernel_size=3, stride=2)
        self.maxpool_2x2s2 = torch.nn.MaxPool2d(kernel_size=2, stride=2)
        self.maxpool_3x3s1 = torch.nn.MaxPool2d(kernel_size=3, stride=1)
        self.maxpool_3x3s2 = torch.nn.MaxPool2d(kernel_size=3, stride=2)

    def forward(self, x):
        x1 = self.avgpool_2x2s2(x)
        x2 = self.avgpool_3x3s2(x)

        x3 = self.maxpool_2x2s2(x)
        x4 = self.maxpool_3x3s1(x)
        x5 = self.maxpool_3x3s2(x)
        return x1, x2, x3, x4, x5

class GlobalPooing(torch.nn.Module):
    def __init__(self):
        super(GlobalPooing,self).__init__()
        self.global_avg = torch.nn.AdaptiveAvgPool2d(output_size=(1, 1))
        self.global_max = torch.nn.AdaptiveMaxPool2d(output_size=(1, 1))
    def forward(self, x):
        x1 = self.global_avg(x)
        x2 = self.global_max(x)
        return x1, x2

已经测试过输出是一致的。  各个算子的速度普遍快了2倍以上

### CODE REVIEW

**Comment by shitouren1994** [comment]
change file不对
